contexts/NotificationsContext.tsx

```diff
@@ -4,28 +4,35 @@ import Notifications from '@/components/Notifications';
 export type TxNotification = {
   id: string;
   hash: `0x${string}`;
-  functionName: string;
+  functionName?: string;
   txStatus: 'pending' | 'success' | 'reverted';
-  chainId: number;
+  chainId?: number;
```

Comment:
should always have chainId

Reply:
In `useEasyWrite` we use `config.request.chainId` which has a type of `number | undefined`.